.github/workflows/merge-main-into-staging.yml

```diff
@@ -0,0 +1,21 @@
+name: Merge main into staging
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
consider using `ref: staging` instead of `git checkout staging` later.

We might also be able to get away with `fetch-depth: 1` (for checkout speed), if we later do a merge from main as `git pull origin main`

Reply:
will use `ref: staging`